fdb-record-layer-lucene/src/test/java/com/apple/foundationdb/record/lucene/LuceneIndexTest.java

```diff
@@ -857,8 +1103,7 @@ private static JoinedRecordTypeBuilder joinedMetadataHook(@Nonnull final RecordM
         final JoinedRecordTypeBuilder joined = metaDataBuilder.addJoinedRecordType("luceneJoinedPartitionedIdx");
         joined.addConstituent("complex", "ComplexDocument");
         joined.addConstituent("simple", "SimpleDocument");
-        joined.addJoin("simple", field("group"), "complex", field("group"));
-        metaDataBuilder.addIndex("SimpleDocument", "simple$group", field("group"));
```

Comment:
I think this was just unnecessary.
I commented it out to see why it was required, and nothing broke, so I just left it off.